setup.py

```diff
@@ -51,7 +51,8 @@ def findsome(subdir, extensions):
         'six>=1.8.0',
         'tqdm',
         'paramiko',
-        'cryptography',
+        'cryptography>=1.5',
+        'enum34',
```

Comment:
what are we using from it?  isn't this one specifically for python2 (backport from python3.4)

Reply:
I put this in because when I build niceman on Debian Jessie, I get this error:

Traceback (most recent call last):
  File "/usr/local/bin/niceman", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3019, in <module>
    @_call_aside
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3003, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3032, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 657, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 670, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 854, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (cryptography 0.6.1 (/usr/lib/python2.7/dist-packages), Requirement.parse('cryptography>=1.5'), set(['paramiko']))

Reply:
I would recommend to avoid installing niceman (unless in a test container) system wide on your development box, all kinds of mix ups could happen causing lots of grief -- virtualenv's should be used.  But may be you were referring to the testing container environment?
BTW, nothing in above traceback points to enum34...

Reply:
Right, sorry, should have said Debian Jessie *in a Docker container*. I have been spinning up containers to do "niceman retrace" runs in various OS environments and with various packages to create spec files. I then use those spec files in "niceman install" runs for development.

I am using miniconda on my laptop to create my Python environments. That's where the above 

+@skip_if(not lexists(opj(dirname(__file__)[:-25], sep, '.git')))

is coming into play...